hpms/src/test/java/gov/cms/ab2d/hpms/service/HPMSAuthServiceTest.java

```diff
@@ -38,6 +38,8 @@
     public void tokenTest() {
         assertNotNull(authService);
 
+        authService.cleanup();
```

Comment:
This is hacky, it already is run after every tests.  Let's discuss whether this is needed and figure out a better way.

Reply:
I think I figured it out. Its MockedFetchTest uses indirectly authservice